### ISSUE TITLE
CPendingDeprecationWarning: The broker_connection_retry configuration setting will no longer determine whether broker connection retries are made during startup in Celery 6.0 and above

### ISSUE THREAD

**Issue by agusmakmun** [cookiecutter/cookiecutter-django#5085]
## Description

I found this deprecation warning for the celery;

```
CPendingDeprecationWarning: The broker_connection_retry configuration setting will no longer
determine whether broker connection retries are made during startup in Celery 6.0 and above.
If you wish to retain the existing behavior for retrying connections on startup,
you should set broker_connection_retry_on_startup to True.
```

https://docs.celeryq.dev/en/stable/userguide/configuration.html#broker-connection-retry

As the newest celery almost using that version;

https://github.com/cookiecutter/cookiecutter-django/blob/b8b470c796c2c91325e9f7349497d2dc148262a1/%7B%7Bcookiecutter.project_slug%7D%7D/requirements/base.txt#L18-L20

## Solution

And found the solution at https://stackoverflow.com/a/78088996/6396981

Where we can utilize the following settings to silence the warning by putting the below code into `celery.py`.

```python
celery_app.conf.broker_connection_retry_on_startup = True
```

or `settings.py`;

```python
CELERY_BROKER_CONNECTION_RETRY_ON_STARTUP = True
```

Just wondering to add this into cookie-cutter-django

**Comment by foarsitter** [comment]
Why do you want the retry on startup behavior?

If I make a mistake in the broker configuration, Celery keeps trying to connect (unsuccessfully, of course). If I'm not paying attention to this, I see a healthy-looking process running. In the new situation, I see an exit(1), which I prefer.

**Comment by browniebroke** [comment]
Agreed, I think this is fine and I don't think we should change anything.

**Comment by browniebroke** [comment]
After further review, closing again as won't fix
